src/TimeService/timeservice-api.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TIMESERVICE_API_H
+#define TIMESERVICE_API_H
+
+#include "timeservice-utils.h"
+
+#define INIT_TIMERS()
+#define ADJUST_ENDIANESS(x)
+
+uint64_t get_clock();
+
+#ifdef RTEMS6_TARGET
+#include "timeservice-rtems.h"
+#endif
+
+#ifdef GENERIC_LINUX_TARGET
+#include "timeservice-linux.h"
+#endif
+
+void TimeService_Startup(void);
+
+
+void TimeService_CfsTimestampCmp(const CfsTimestamp *t1, const CfsTimestamp *t2, int *result);
```

Comment:
can't we just return the value result, do we have to have void function everywhere?

Reply:
We can it's just architecture based on the interfaces in taste but in the API I can change it

Reply:
I changed the convention wherever it was possible